metrics/influx/influx.go

```diff
@@ -0,0 +1,213 @@
+package metrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prebid/prebid-cache/config"
+	"github.com/rcrowley/go-metrics"
+	"github.com/sirupsen/logrus"
+	"github.com/vrischmann/go-metrics-influxdb"
+)
+
+var TenSeconds time.Duration = time.Second * 10
+
+//	Object definition
```

Comment:
Please remove this comment. It is associated with the InfluxMetrics by godocs.

Reply:
done